specs/http_auth_SUITE.erl

```diff
@@ -0,0 +1,80 @@
+-module(http_auth_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("deliverly/include/deliverly.hrl").
+-include_lib("deliverly/include/log.hrl").
+-include_lib("deliverly/include/priv.hrl").
+
+-import(test_utils, [
+  gun_open/1,
+  auth_headers/2,
+  post/3
+]).
+
+-compile(export_all).
+
+init_per_suite(Config) ->
+  ulitos_app:set_var(?APP, redis_database, 7),
+  ulitos_app:set_var(?APP, api_user, <<"admin">>),
+  ulitos_app:set_var(?APP, api_pass, list_to_binary(ulitos:random_string(8))),
+  lager:start(),
+  deliverly:start(),
+  ulitos_app:ensure_started(gun),
+  [{url, "/api/auth/token"}, {port, 8081} | Config].
+
+end_per_suite(_) ->
+  ulitos_app:set_var(?APP, api_user, <<"">>),
+  ulitos_app:set_var(?APP, api_pass, <<"">>),
+  redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),
```

Comment:
Line 28 is too long:   redis_cli:q(["EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))",  0,  "*"]),.